Core/NsvController.cs

```diff
@@ -0,0 +1,88 @@
+using System;
+using System.Collections.Generic;
+using SliceVisualizer.Configuration;
+using UnityEngine;
+using UnityEngine.UI;
+using Zenject;
+
+namespace SliceVisualizer.Core
+{
+    internal class NsvController : IInitializable, ITickable, IDisposable
+    {
+        private readonly PluginConfig _config;
+        private readonly BeatmapObjectManager _beatmapObjectManager;
+        private readonly NsvSlicedBlock.Pool _slicedBlockPool;
```

Comment:
OK, please hear me out on this: using a memory pool here is less efficient than having a static pool of objects.
Spawning/Despawning/memory management will take more time during the game than just spawning these 12 objects.